### PR TITLE
Feat/branding UI tests

### PR DESCRIPTION
**Pull request by andrewleith**
# Summary | Résumé

This PR adds UI tests for the branding features

### Branding settings
- [x] Passes a11y and HTML validation
- [x] Current branding can be seen
- [x] Current branding can be previewed

### Choose GOC brandings
- [ ] Passes a11y and HTML validation
- [x] Cannot submit when no selection made
- [x] English-first logo is saved when selected
- [x] French-first logo is saved when selected

### Choose from pool
- [x] Passes a11y and HTML validation
- [x] No selection made
- [x] Branding can be previewed
- [x] Branding can be saved
- [x] Org has no logos

### Preview Branding
- [x] Passes a11y and HTML validation
- [x] GOC english-first brand can be previewed
- [x] GOC french-first brand can be previewed
- [ ] Custom logo can be previewed

### Branding request tests
- [x] Passes a11y and HTML validation
- [x] Loads request branding page
- [x] Disallows submission when there is no data
- [x] Disallows submission when there is no image 
- [x] Disallows submission when there is no brand name 
- [x] Only allows pngs
- [x] Allows submission when all valid data provided 
- [x] Displays branding preview 
- [ ] Rejects malicious files

### CODE REVIEW

File: tests_cypress/cypress/e2e/admin/branding/edit-branding.cy.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+            cy.get('img').last().should('have.attr', 'src').then(src => {
+                expect(src).to.include('gc-logo-fr.png');
+            });
+        })
+    });
+
+    it('Loads edit-branding when choose logo is clicked', () => {
+        cy.visit(config.Hostnames.Admin + `/services/${config.Services.Cypress}/branding`);
+        BrandingSettings.ChooseDifferentLogo();
+        cy.get('h1').contains('Change your branding').should('be.visible');
```
Comment (andrewleith) [inline tests_cypress/cypress/e2e/admin/branding/edit-branding.cy.js:38]:
```suggestion
        cy.get('h1').contains('Change your logo').should('be.visible');
```

File: tests_cypress/cypress/e2e/admin/branding/edit-branding.cy.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    });
+
+    it('Cannot submit when no selection was made', () => {
+        EditBranding.Submit();
+        cy.contains('span', 'You need to choose an option').should('be.visible');
```
Comment (andrewleith) [inline tests_cypress/cypress/e2e/admin/branding/edit-branding.cy.js:43]:
```suggestion
        cy.contains('span', 'Change your logo').should('be.visible');
```

File: tests_cypress/cypress/e2e/admin/branding/edit-branding.cy.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+    it('Navigates to branding pool when select another logo from test link is clicked', () => {
+        EditBranding.ClickBrandPool();
+        cy.contains('h1', 'Select alternate logo').should("be.visible");
+    })
+
+    it('Navigates to edit branding when go back is clicked from the branding pool page', () => {
+        EditBranding.ClickBrandPool();
+        BrandingSettings.GoBack();
+        cy.get('h1').contains('Change your branding').should('be.visible');
```
Comment (andrewleith) [inline tests_cypress/cypress/e2e/admin/branding/edit-branding.cy.js:68]:
```suggestion
        cy.get('h1').contains('Change your logo').should('be.visible');
```

**Review by andrewleith — COMMENTED** [review]
A couple of changes needed for content/design changes that have occurred. 

For the review pool tests, the `Saves the selected custom logo` test is failing and I noticed that one of the components returns all the radio options as an array and then tries to click it, perhaps you meant to select just one?

```
 AvailableLogoRadios: () => cy.get('input[id*="pool_branding-"]'),
```

File: tests_cypress/cypress/e2e/admin/service-settings.cy.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+        `/services/${config.Services.Cypress}/service-settings`,
+    );
+  });
+
+  it("Loads service settings page", () => {
+    cy.get("h1").contains("Settings").should("be.visible");
+  });
+
+  it("Loads branding settings page", () => {
+    ServiceSettingsPage.ClickChangeEmailBrandingLink();
```
[the remark this answers is not in the document]
Comment (andrewleith) [reply tests_cypress/cypress/e2e/admin/service-settings.cy.js:26]: Good catch! Why can't my IDE do that?? 😭

**Review by whabanks — APPROVED** [review]
LGTM! Nice work getting these working!

**Pull request merged**